projects/plugins/jetpack/extensions/blocks/business-hours/business-hours.php

```diff
@@ -12,33 +12,16 @@
 use Automattic\Jetpack\Blocks;
 use Jetpack_Gutenberg;
 
-const FEATURE_NAME = 'business-hours';
-const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
-
 /**
  * Registers the block for use in Gutenberg
  * This is done via an action so that we can disable
  * registration if we need to.
  */
 function register_block() {
 	Blocks::jetpack_register_block(
-		BLOCK_NAME,
+		__DIR__,
```

Comment:
I had to replace `Blocks::get_path_to_block_metadata( __DIR__ )` (the path to the `block.json` file in the build folder) with `__DIR__` (the path to the `block.json` file in the source folder). The former implementation worked locally but made the test jobs in the CI pipeline fail: files in the build folder don't seem to exist at the time the tests are run. Please let me know if you have any insights about this.

The `block.json` files in the build folder include paths to scripts and stylesheets. Currently, block assets are registered and enqueued manually, so it doesn't matter which `block.json` we use for registration. But eventually, we'll need to use the one in the build folder.